tiledb/sm/query/writer.h

```diff
@@ -880,6 +1037,29 @@ class Writer {
       const std::string& attribute,
       FragmentMetadata* frag_meta,
       const std::vector<Tile>& tiles) const;
+
+  // TODO: remove
```

Comment:
If this is intended to be checked in, could you expand this comment to explain why it should be removed in a future patch?

Reply:
This will be removed in my next PR over the next couple of days, but I will add a comment.